packages/flutter_driver/lib/src/driver.dart

```diff
@@ -15,13 +15,37 @@ import 'health.dart';
 import 'message.dart';
 import 'timeline.dart';
 
+enum TracingCategory {
```

Comment:
Looks like these are referred to as "timeline stream" rather than "tracing category".